### PR TITLE
fare_InformativeBestPricingWithouthPNR add optionDetail to pricingOptionKey

### PR DESCRIPTION
**Pull request by LeoTravel**
I've implemented a very raw param that allow you to add an optionDetailAttribute to an overrideOption.

```php
new FareInformativeBestPricingWithoutPnrOptions([
        'pricingOptions' => new PricingOptions([
            'overrideOptions' => ['RU'],
            'overrideOptionsWithCriteria' => [
                [
                    'key' => 'SBF',
                    'optionDetail' => '1'
                ]
            ]
        ])
    ]);
```

xml without optionDetailAttribute:
```xml
<ns1:pricingOptionGroup>
	<ns1:pricingOptionKey>
		<ns1:pricingOptionKey>RU</ns1:pricingOptionKey>
	</ns1:pricingOptionKey>
</ns1:pricingOptionGroup>
````

xml with optionDetailAttribute
```xml
<ns1:pricingOptionGroup>
	<ns1:pricingOptionKey>
		<ns1:pricingOptionKey>RU</ns1:pricingOptionKey>
	</ns1:pricingOptionKey>
</ns1:pricingOptionGroup>
<ns1:pricingOptionGroup>
	<ns1:pricingOptionKey>
		<ns1:pricingOptionKey>SBF</ns1:pricingOptionKey>
		<optionDetail>
			<criteriaDetails>
				<attributeType>1</attributeType>
			</criteriaDetails>
		</optionDetail>
	</ns1:pricingOptionKey>
</ns1:pricingOptionGroup>
````

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,4 +39,4 @@
       "php": "5.4"
     }
   }
-}
```
Comment (DerMika) [inline composer.json]: Can you please undo this change?

**Comment by gabrythe23** [comment]
i've fix composer.json, implemented a very simple coverage function for overrideOptionsWithCriteria and throwed an exception when overrideOptionsWithCriteria isn't empty in ```PricePNRWithBookingClass12```

**Comment by LeoTravel** [comment]
everything seems okay now, but we still receive some errors from unittest, don't know why.

**Comment by DerMika** [comment]
The unittest seems to be working now? 

It's just the code coverage that is acting up. It says coverage decreased, but it seems like a rounding error at coveralls.

I'm going to merge your PR.

**Comment by DerMika** [comment]
See #216

**Comment by DerMika** [comment]
Thanks a lot for your contribution!
